subprojects/common/graphite/src/main/kotlin/com/avito/android/graphite/GraphiteSender.kt

```diff
@@ -31,9 +31,9 @@ interface GraphiteSender {
                     val dos = DataOutputStream(socket.getOutputStream())
                     dos.writeBytes(message)
                 }
-                logger.debug("$tag: $message")
+                logger.info(message)
```

Comment:
It seems overkill to duplicate all events in logs. We might have thousands of them in one build.

Reply:
yep, removed